pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -17,9 +17,13 @@
 let
   py = python3.override {
     packageOverrides = self: super: {
-      django = super.django_3;
```

Comment:
`fetchpatch` can be removed from the inputs

Reply:
Done.